doc/en/user/source/extensions/vectortiles/options.rst

```diff
@@ -0,0 +1,108 @@
+.. _vectortiles.options:
+
+Vector Tiles Generation Options
+-------------------------------
+
+The content of the vector tiles generated by this output format is driven by the
+style used in the GetMap/GetTile request.
+
+The style can be pretty basic, the vector tile output format considers only the following:
```

Comment:
The style can be pretty basic. The vector tile output format considers only the following: